doc/configuration.md

```diff
@@ -37,6 +37,7 @@ The Ignition configuration is a JSON document conforming to the following specif
       * **device** (string): the absolute path to the device. Devices are typically referenced by the `/dev/disk/by-*` symlinks.
       * **format** (string): the filesystem format (ext4, btrfs, or xfs).
       * **_create_** (object): contains the set of options to be used when creating the filesystem. A non-null entry indicates that the filesystem shall be created.
+        * **_allowExisting_** (boolean): whether or not the create operation shall be skipped if the target filesystem is already formatted with the desired filesystem type.
```

Comment:
Yeah, the mix of force and this option is weird. I would expect force to always format rendering this option redundant and normal behavior to not introspect fs type, just whether a fs exists. That's how tools usually work.

So what you really should have is a enum replacing force and this option:

overwrite: never, unknown, always.

Name of 'unknown' subject to debate but that's the mode where wrong filesystems are replaced.